### DISCUSSION TITLE
theming label of inputs

### DISCUSSION THREAD

**Discussion by hyusetiawan** [mantinedev/mantine#972] [Q&A] [answered]
I am trying to theme label of inputs, such as
```
<Select label="My Select" />
```
or 
```
<TextInput label="My Text" />
```
I would like to bold the label, I see there is a guide for theming via `<MantineProvider />`, but I could not for the life of me figure out the right config, is there an example I can follow?

**Comment by rtivital** [comment, marked as answer]
https://codesandbox.io/s/eloquent-smoke-je8x9v?file=/src/App.tsx

**Reply by hyusetiawan** [reply]
apologies, I should have been clearer, I am attempting to theme it instead of overriding a specific instance of a component. The closest I see is: 
https://mantine.dev/theming/styles-api/#:~:text=Button%3A%20(theme%2C%20params%3A%20ButtonStylesParams)%20%3D%3E%20(%7B
but unfortunately, I don't know what to put to style the label specifically, because the typing is of Record<string, CSS>

**Reply by rtivital** [reply]
https://codesandbox.io/s/distracted-tess-qj0sr0?file=/src/App.tsx

**Reply by hyusetiawan** [reply]
@rtivital oh dang, thank you so much! yeah, I should have guessed that
how did you find this particular `property`? Is there a guide to navigating what properties are available?

**Reply by rtivital** [reply]
what property?

**Reply by hyusetiawan** [reply]
the property `label` inside `TextInput`, like where can I see what other property names are there inside `TextInput`?

**Reply by rtivital** [reply]
https://mantine.dev/theming/styles-api/

**Reply by reza-ebrahimi** [reply]
@rtivital Why major/(all?) components in mantine by default is set to bold font weight instead of normal? Like inputs and buttons.

**Reply by rtivital** [reply]
These are the default styles, I've chosen. If you do not like them, you can easily override them with the Styles API.
